tests/unit/test_changelog.py

```diff
@@ -180,6 +201,19 @@ def test_extract_releases(github_payload, release_dict, mocker):
     ]
 
 
+def test_extract_releases_custom_graphql_url(github_payload, release_dict, mocker):
+    mocker.patch(
+        "sphinx_github_changelog.changelog.github_call", return_value=github_payload
+    )
+    assert changelog.extract_releases(
+        owner_repo="a/b",
+        token="token",
+        graphql_url="https://git.privaterepo.com/graphql",
+    ) == [
+        release_dict,
+    ]
```

Comment:
(We're not exactly testing that the `graphql_url` is used, but it could be good enough)